pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -200,31 +546,91 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Method for Computing FIM \n",
+    "\n",
+    "This method computes an FIM-based MBDoE optimization problem with zero degrees of freedom."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fisher Information Matrix (FIM)\n",
+    "The FIM measures the information content for the unknown parameters $\\theta$ from the model output $y_i$ given that:\n",
+    "\n",
+    "$$ y_i = f(\\psi_i, \\theta) $$\n",
+    "\n",
+    "where $\\psi$ is a design vector from a DAE system.\n",
+    "\n",
+    "In order the quantify the uncertainty of the estimated parameters for parameter estimation, consider the covariance matrix for the parameters:\n",
+    "\n",
+    "$$V(\\hat{\\theta},\\psi) = \\left[\\sum_{r}^{N_{r}}\\sum_{r}^{N_{r}} \\tilde{\\sigma}_{(r,r')}Q_{r}^{T}Q_{r'}+V_{\\theta}(\\hat{\\theta})^{-1}\\right]^{-1}$$\n",
+    "\n",
+    "where:\n",
+    "* $\\hat{\\theta}$: estimated parameters.\n",
+    "* $\\tilde{\\sigma}$: element in the inverse of the observational covariance matrix.\n",
+    "* $r,$ $r'$: measurements.\n",
+    "* $Q$: dynamic sensitivity.\n",
+    "* $V_{\\theta}$: prior information.\n",
+    "* $N_r$: number of measurements\n",
+    "\n",
+    "The inverse of $V$ estimates the FIM such that:\n",
+    "\n",
+    "$$V(\\hat{\\theta},\\psi) = [M(\\hat{\\theta},\\psi)]^{-1}$$\n",
+    "\n",
+    "For sequential design of experiments, consider prior information such that after $N_e$ experiments, the FIM is calculated by:\n",
+    "$$M= \\sum_{k=1}^{N_e-1}M_k+M_{N_e}(\\hat{\\theta},\\psi_{N_e}) = K+M_{N_e}(\\hat{\\theta},\\psi_{N_e})$$\n",
+    "\n",
+    "where:\n",
+    "* $N_e - 1$: previous experiments\n",
+    "* $K$: constant matrix encoding information from all $N_e - 1$\n",
+    "\n",
+    "**Key Takeaways**:\n",
+    "In regards to parameter estimation, a **large** FIM value denotes **more** information about $\\theta$ is gained from the model."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Method: Compute FIM \n",
+    "### Model-Based Design of Experiments\n",
+    "The objective of MBDoE changes the conditions of one or more experiments based on a specific purpose such as:\n",
+    "\n",
+    "1. Model identifaction\n",
+    "    * Discriminates between possible models while omitting ineadequate models.\n",
+    "2. Parameter estimation\n",
+    "    * Improves parameter estimation precision.\n",
     "\n",
-    "This method computes an MBDoE optimization problem with no degrees of freedom."
+    "**Key Takeaways:**\n",
+    "Given an estimate for an unknown parameter and one or more mathematical models, MBDoE:\n",
+    "1. Determines a set of experimental conditions to maximize the precision of the unknown model parameters.\n",
+    "2. Discriminates between the given models.\n",
+    "3. Or both (1) and (2)."
```

Comment:
After the list add: Currently, Pyomo.DoE supports MBDoE for parameter precision.